RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,3 @@
+# Client-Java
+
+ { Please add the release notes here }
```

Comment:
Are you planning to add the installation instruction here too? I think we should show how to declare the dependency (`<groupId></>`, `<artifactId></>`, `<version></>`) in their pom.xml (like how we show in docs)

Reply:
fixed